monitoring/README.md

```diff
@@ -1,112 +1,66 @@
+pyroscope-alloy-0                                     2/2     Running   0          90s
+```
 
-a) Prometheus Server: edit the file roles/prometheus/templates/prometheus-config.yaml.j2
+2. Configure the Prometheus and Pyroscope to scrape your KubeStellar spaces:
 
+```bash
+$ ./configure-metrics-tools.sh
 ```
-global:
-  evaluation_interval: 5s
-  external_labels:
-    env: dev
-  scrape_interval: 30s
-scrape_configs:
-- job_name: mailbox-controller
-  scrape_interval: 15s
-  metrics_path: /metrics
-  static_configs:
-  - targets:
-    - '<host-ipAddress>:10203'
-
-- job_name: node-exporter
-  scrape_interval: 15s
-  metrics_path: /metrics
-  static_configs:
-  - targets:
-    - '<host-ipAddress>:9100'
-
-- job_name: kcp
-  scrape_interval: 15s
-  scheme: https
-  metrics_path: /metrics
-  tls_config:
-    insecure_skip_verify: true
-  static_configs:
-  - targets:
-    - '<host-ipAddress>:6443'
 
+Optionally, check the prometheus service monitor objects:
+
+```bash
+kubectl -n ks-monitoring get servicemonitor
+NAME                                                 AGE
+ks-controller-manager                                6h24m
+ks-transport-controller                              6h24m
+prometheus-kube-prometheus-apiserver                 21h
+prometheus-kube-prometheus-coredns                   21h
+prometheus-kube-prometheus-kube-controller-manager   21h
+prometheus-kube-prometheus-kube-etcd                 21h
+prometheus-kube-prometheus-kube-proxy                21h
+prometheus-kube-prometheus-kube-scheduler            21h
+prometheus-kube-prometheus-kubelet                   21h
+prometheus-kube-prometheus-operator                  21h
+prometheus-kube-prometheus-prometheus                21h
+wds-apiserver                                        6h24m
 ```
 
 
-b) Prometheus Agent: edit the file roles/prometheus/templates/prometheus-agent-config.yaml.j2
+3. Connect to Grafana UI: 
 
-```
-# my global config
-global:
-  scrape_interval: 15s
-  evaluation_interval: 15s 
-scrape_configs:
-  - job_name: "prometheus-agent"
-
-    static_configs:
-      - targets: ["{{ ansible_default_ipv4.address }}:<port_number>"]
-remote_write:
-  - url: "http://<prometheus-server-ip-address>:<port_number>/api/v1/write"
-```
+a) Port-forward Grafana to localhost, by using the kubectl command:
 
-
-3. Install Prometheus server, grafana and node exporter to the KCP server host using the following playbook:
-
-```
-- hosts: kcp-server
-  remote_user: ubuntu
-  become: yes
-  gather_facts: yes
-  connection: ssh
-  tasks:
-  roles:
-    - node-exporter
-    - prometheus
-    - grafana
+```bash
+$kubectl port-forward -n ks-monitoring svc/grafana 3000:80
 ```
 
-```
-ansible-playbook -i hosts monitoring-kcpServer.yaml
-```
+b) In a browser, go to the Grafana server at http://localhost:3000 and on the left-hand side, go to Configuration > Data sources.
+
+<img src="images/grafana-data-sources.png" width="60%" height="60%" title="grafana-data-sources">
 
```

Comment:
Did some content get dropped here? What I am I supposed to do after opening the Data sources panel?

Reply:
Fixed. I added the instructions on how to import the grafana dashboards and visualize the data